### ISSUE TITLE
gitbook.com: TypeError: Invalid Version: >=2.3.0

### ISSUE THREAD

**Issue by Vdragon** [GitbookIO/gitbook#910]
Hi, recently I found my GitBook build fails with `TypeError: Invalid Version: >=2.3.0` message, according to [Configuration | Documentation](https://help.gitbook.com/format/configuration.html) gitbook 2.3.1 should build a GitBook with a "gitbook": >=2.3.0 pair in `book.json`

```
Downloading source
Extracting source
GitBook version is 2.3.1
Tweaking book.json
Installing plugins

TypeError: Invalid Version: >=2.3.0
```
repo: https://github.com/Vdragon/Hello_Git_World_

**Comment by mattkanwisher** [comment]
Same thing
warn: gitbook version specified in your book.json might be too strict for future patches, "2.x.x" is more adequate

TypeError: Invalid Version: 2.x.x

**Comment by SamyPesse** [comment]
I'm working on a fix, and I'll publish 2.3.2

**Comment by Vdragon** [comment]
:+1:
